client/src/components/categorical/value.js

```diff
@@ -348,6 +356,13 @@ class CategoryValue extends React.Component {
               <form
                 onSubmit={e => {
                   e.preventDefault();
+                  if (
+                    (category.categoryValues.indexOf(editedLabelText) > -1 &&
```

Comment:
much like we did in categorical.js and category.js, this error check,  and the corresponding error message (line 416) should be factored into a helper function for clarity.

Reply:
I think you missed this check - shouldn't this call the category error check?